packages/vulcan-forms/lib/components/FormComponent.jsx

```diff
@@ -117,27 +117,20 @@ class FormComponent extends Component {
   getValue = props => {
     let value;
     const p = props || this.props;
-    const {
-      document,
-      currentValues,
-      defaultValue,
-      emptyValue,
-      isEmptyValue,
-      mergeValue,
-    } = p;
+    const { document, currentValues, defaultValue } = p;
     // for intl field fetch the actual field value by adding .value to the path
     const path = p.locale ? `${this.getPath(p)}.value` : this.getPath(p);
     const documentValue = get(document, path);
     const currentValue = get(currentValues, path);
     const isDeleted = p.deletedValues.includes(path);
 
     if (isDeleted) {
-      value = emptyValue;
+      value = '';
     } else {
       value = mergeValue({ currentValue, documentValue, ...p });
```

Comment:
@SachaG i've kept `mergeValue` as a function to keep it more encapsulated, but now it cannot be passed as prop.